### ISSUE TITLE
Ubuntu Stak Hanging

### ISSUE THREAD

**Issue by Loerb01** [fireice-uk/xmr-stak#773]
Please provide as much as possible information to reproduce the issue.

# Basic information
Intel Celeron
Radeon Rx 580(Mixed)

# Stability issue
  - Is the CPU or GPU overclocked? No
  - Is the Main memory of the CPU or GPU undervolted? No

Newest version of Ubuntu as of 12/26/17 then updated on 12/31. Amd Gpu pro compute drivers. Memory manually pinned, huge pages set to 128, runs from 30min up to 12 hr then hangs. Computer is fine just Stak crashes. No crash, reboot or error. Stak cannot recompile video cards without whole system reboot.

**Comment by mvpel** [comment]
Aren't the ROCm drivers for Ubuntu only available on 16.04, not 17?

**Comment by Loerb01** [comment]
@mvpel Why do I need them and how would I go about getting them?

**Comment by BisWang** [comment]
Same problem with Ubuntu16.04 with H81 pro BTC MB and 6 750Ti

**Comment by joshso** [comment]
Seeing similar on Win 10 / i5 stock clock CPU / 2x Vega 64's using 2.2.0. Made it 12 hours today, found the app window open but unresponsive, web server process had stopped responding, OS was responsive. No errors listed and no crash.

Killed the process, reset everything and it ran for 20 minutes before repeating - exact same behavior. This rig is brand new and the 12 hours is the longest stretch I've had; been tuning and fixing issues as I find them. I opened Task Manager and the system appeared to still be mining before I killed the process, judging by the 3 CPU cores I have working and a gpu memory graph.

Next time will check my pool website to see if it still reports activity, and can bump verbose_level back up from 1 where I have it currently. It could be that the web server process stops responding and that just makes me think the miner stopped too.
